tests/tests_ansible.yml

```diff
@@ -333,13 +333,34 @@
           zone: customzone
           state: present
           permanent: yes
+          immediate: yes
         register: result
 
       - name: assert firewalld custom zone
         assert:
           that:
             - result is changed
 
+      - name: Add masquerade to custom zone
+        firewall_lib:
+          masquerade: yes
+          zone: customzone
+          state: enabled
+          permanent: yes
+          immediate: yes
+        register: result
+        failed_when: result.failed or not result.changed
```

Comment:
should use an ansible test for this: https://docs.ansible.com/ansible/2.9/user_guide/playbooks_tests.html#task-results
`failed_when: result is failed or not result is changed`

Reply:
hmm - looking at some of the existing tests - they all use the `result.failed` style instead of `result is failed` - I think you should not go back and change all of them, but if that code is changed in the future for some reason, it should also change the tests to use the `result is failed` style.